webapp_utils/errata.py

```diff
@@ -142,23 +148,25 @@ def _fill_errata(self, search):
             return errata_list
         db_connection = self.db_pool.get_connection()
         with db_connection.get_cursor() as cursor:
-            for errata in search:
-                errata = '%' + errata + '%'
-                cursor.execute("""select name from errata
-                                  where name like '%s'
-                               """ % errata)
-                query = cursor.fetchall()
-                for item in query:
-                    errata_list.append(item[0])
+            errata = "%" + search + "%"
+            cursor.execute("""select name from errata
+                              where name like %s""", (errata,))
+            query = cursor.fetchall()
+            for item in query:
+                errata_list.append(item[0])
         return errata_list
 
     def _errata_exists(self, errata):
```

Comment:
This will cause us to run N+1 queries for N erratas. In the end, this method is redundant, since process_list in `errata.py` performs a query, which returns all erratas.

Reply:
Fixed by [PR 613](https://github.com/RedHatInsights/vmaas/pull/613)